charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -46,6 +46,9 @@ securityContext:
 nodeSelector:
   kubernetes.io/arch: amd64
 
+affinity: {}
```

Comment:
Could you elaborate on the usecase for affinity on Daemonsets? What would be a practical example on when to use that?